### DISCUSSION TITLE
Is the package safe to use for Android only while build failing?

### DISCUSSION THREAD

**Discussion by kobe-ra** [tekartik/sqflite#681] [Q&A]
hey, so i was just about to start using this package after looking at it for several months now, but the [build failing tag](https://travis-ci.org/github/tekartik/sqflite) looks pretty scary while i see there is constant work being done on this package.

So is the package safe to use, is this just a temporary issue or is sth actually wrong?

**Comment by alextekartik** [comment]
travis seems deprecated now but github actions (https://github.com/tekartik/sqflite/actions) are also ran on windows/mac/linux on flutter stable/beta/dev on push/merge and every week. I think that setMockMethodCallHandler has been removed - that breaks some unit tests - and an alternate solution should be used but it is not available on flutter stable yet. so build should be ok on stable and fail during unit test on beta and dev. Otherwise sqflite should work on all flutter branches.

I monitor CI issues but sometimes I cannot fix the build in the beta/dev without breaking the stable branch. Of course CI success/failure does not ensure that the package will work on Android however I have not heard of recent failure and I use it myself in production.
